sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -2392,5 +2392,13 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
         }
       }
     }
+    test("load command invalid path validation ") {
```

Comment:
@sujith71955 according to Jenkins, there's a whitespace at end of line

Reply:
updated it. In my local build it was working fine. Thanks for the feedback